backend/src/api/likeApi.ts

```diff
@@ -0,0 +1,14 @@
+import * as express from 'express';
+import ctrlLikes from '../controllers/likesController';
+
+const router = express.Router();
+
+router.post('/like', ctrlLikes.create);
+
+router.post('/:postId/like', ctrlLikes.create);
```

Comment:
Do not duplicate endpoints (/like). Stay RESTful and operate with POST /likes including `postId` in request body, considering that Like is standalone entity with separate schema with postId: ref to Post